detect/detect_test.go

```diff
@@ -114,6 +114,72 @@ func TestDetect(t *testing.T) {
 				},
 			},
 		},
+		{
+			cfgName: "simple",
+			fragment: Fragment{
+				Raw:      `export BUNDLE_ENTERPRISE__CONTRIBSYS__COM=cafebabe:deadbeef;`,
+				FilePath: "tmp.sh",
+			},
+			expectedFindings: []report.Finding{
+				{
+					Description: "Sidekiq Secret",
+					Match:       "BUNDLE_ENTERPRISE__CONTRIBSYS__COM=cafebabe:deadbeef;",
+					Secret:      "cafebabe:deadbeef",
+					File:        "tmp.sh",
+					RuleID:      "sidekiq-secret",
+					Tags:        []string{},
+					Entropy:     2.6098502,
```

Comment:
NOTE1: You may consider these tests redundant or unnecessary since the rule validator already does some testing. However, these were useful for me in getting the secret group right (maybe something to add to the validator?) and also just helped me build confidence in more complex examples like you see on line 142 below, although ultimately I think the validator would have caught that one too.

NOTE2: I must admit, for the expected values for Entropy and some of the StartColumn and EndColumn values in these tests, I just copied from the failed test results. Let me know if you think those need to be manually verified.